### PR TITLE
Updating node.yml workflow from template

### PR DESCRIPTION
**Pull request by nextcloud-command**
Automated update of the node.yml workflow from https://github.com/nextcloud/.github

### CODE REVIEW

File: .github/workflows/node.yml [hunk 1]
```diff
@@ -39,14 +42,8 @@ jobs:
 
       - name: Install dependencies & build
         run: |
-          make clean
           npm ci
           npm run build --if-present
-          npm run build:package
```
Comment (max-nextcloud) [inline .github/workflows/node.yml:45]: I'd like to ensure the package build does not break.
We could use a different workflow for that though.

File: .github/workflows/node.yml [hunk 2]
```diff
@@ -39,14 +42,8 @@ jobs:
 
       - name: Install dependencies & build
         run: |
-          make clean
```
Comment (max-nextcloud) [inline .github/workflows/node.yml:42]: If we remove the make clean the js dir will contain the artifacts from the repo, right? So if `npm run build` did nothing the comparison to the repo would still match in the next step.

I'm not sure which of these are changes from the template and which are just cleanup.

File: .github/workflows/node.yml [hunk 3]
```diff
@@ -39,14 +42,8 @@ jobs:
...
           npm ci
           npm run build --if-present
-          npm run build:package
-
-      - name: npm test
-        run: |
-          npm test
```
Comment (max-nextcloud) [inline .github/workflows/node.yml:49]: Do we run these tests elsewhere?

**Review by max-nextcloud — COMMENTED** [review]
I don't quite understand the goal here. @vinicius73 are you aiming to make this match the template more closely? Shall we move the text specific checks to other workflows?

**Comment by vinicius73** [comment]
> I don't quite understand the goal here. @vinicius73 are you aiming to make this match the template more closely? Shall we move the text specific checks to other workflows?

No @max-nextcloud 
It is an automatic PR, and I know it will impact the generation of text package.
Because of that, I've called for you.

**Comment by max-nextcloud** [comment]
> > I don't quite understand the goal here. @vinicius73 are you aiming to make this match the template more closely? Shall we move the text specific checks to other workflows?
> 
> No @max-nextcloud It is an automatic PR, and I know it will impact the generation of text package. Because of that, I've called for you.

Ahh... now i get it. Thanks. then we should only add the permissions i think. I'll fix this.

**Review by max-nextcloud — APPROVED** [review]
Okay... let's just add what was added upstream.

**Pull request merged**